packages/shared/src/components/notifications/NotificationItem.tsx

```diff
@@ -0,0 +1,79 @@
+import classNames from 'classnames';
+import React, { ReactElement } from 'react';
+import { Notification } from '../../graphql/notifications';
+import { useDomPurify } from '../../hooks/useDomPurify';
+import NotificationItemAttachment from './NotificationItemAttachment';
+import NotificationItemAvatar from './NotificationItemAvatar';
+
+export interface NotificationItemProps
+  extends Pick<
+    Notification,
+    'type' | 'icon' | 'title' | 'description' | 'avatars' | 'attachments'
+  > {
+  isUnread?: boolean;
+  targetUrl?: string;
+}
+
+function NotificationItem({
+  icon,
+  type,
+  title,
+  isUnread,
+  description,
+  avatars,
+  attachments,
+}: NotificationItemProps): ReactElement {
+  const purify = useDomPurify();
+
+  const avatarComponents =
+    avatars?.map?.((avatar) => (
+      <NotificationItemAvatar key={avatar.referenceId} {...avatar} />
+    )) ?? [];
```

Comment:
To filter out the nulls
```suggestion
  const avatarComponents =
    avatars?.map?.((avatar) => (
      <NotificationItemAvatar key={avatar.referenceId} {...avatar} />
    )).filter((avatar) => avatar) ?? [];
```